### PR TITLE
fix: removed the parent padding and instead applied padding individually.

### LINKED ISSUE #3389: [Bug] divider between new page and trash does not stretch to edge of sidebar
### Bug Description

Title says all.

Please make sure that while fixing this, the rest of the sidebar UI doesn't change

### How to Reproduce

1. Open AppFlowy.

### Expected Behavior

The divider should start from the edge of the window and extend to the edge of the sidebar

### Operating System

macOS

### AppFlowy Version(s)

0.3.1

### Screenshots

Screenshot 2023-09-12 at 9 29 24 PM

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by Kritarthsharma**
Fixes #3389 

The divider is now starting from the window and stretching to the edge of the Sidebar without hindering with rest of the Sidebar UI.

Screenshot 2023-10-27 at 5 18 43 PM
Screenshot 2023-10-27 at 5 18 56 PM

---

#### PR Checklist

- [x] My code adheres to [AppFlowy's Conventions](https://docs.appflowy.io/docs/documentation/software-contributions/conventions)
- [x] I've listed at least one issue that this PR fixes in the description above.
- [ ] I've added a test(s) to validate changes in this PR, or this PR only contains semantic changes.
- [x] All existing tests are passing.

### CODE REVIEW

File: frontend/appflowy_flutter/lib/workspace/presentation/home/menu/sidebar/sidebar.dart [hunk 1]
```diff
@@ -94,37 +94,43 @@ class HomeSideBar extends StatelessWidget {
...
+          // user, setting
+          Padding(
+            padding: const EdgeInsets.symmetric(horizontal: 12),
+            child: SidebarUser(user: user, views: views),
+          ),
+          const VSpace(20),
+          // scrollable document list
+          Expanded(
+            child: Padding(
+              padding: const EdgeInsets.symmetric(horizontal: 12),
```
Comment (Xazin) [inline frontend/appflowy_flutter/lib/workspace/presentation/home/menu/sidebar/sidebar.dart:115]: You can extract this padding to a const:
```
const _menuHorizontalInset = EdgeInsets.symmetric(horizontal: 12);
```

File: frontend/appflowy_flutter/lib/workspace/presentation/home/menu/sidebar/sidebar_new_page_button.dart [hunk 2]
```diff
@@ -48,7 +48,10 @@ class SidebarNewPageButton extends StatelessWidget {
       height: 60,
       child: TopBorder(
         color: Theme.of(context).dividerColor,
-        child: child,
+        child: Padding(
+          padding: const EdgeInsets.symmetric(horizontal: 16),
```
Comment (richardshiue) [inline frontend/appflowy_flutter/lib/workspace/presentation/home/menu/sidebar/sidebar_new_page_button.dart:52]: Where did 16 come from?

File: frontend/appflowy_flutter/lib/workspace/presentation/home/menu/sidebar/sidebar.dart (hunk 1 above)
Comment (Kritarthsharma) [reply frontend/appflowy_flutter/lib/workspace/presentation/home/menu/sidebar/sidebar.dart:115]: Sure! Will make the mentioned changes

File: frontend/appflowy_flutter/lib/workspace/presentation/home/menu/sidebar/sidebar_new_page_button.dart (hunk 2 above)
Comment (richardshiue) [reply frontend/appflowy_flutter/lib/workspace/presentation/home/menu/sidebar/sidebar_new_page_button.dart:52]: The original was 12 pixels and then an additional 6. I suppose it should be 18? It seems a bit off when zooming into the screenshot above

Comment (Kritarthsharma) [reply frontend/appflowy_flutter/lib/workspace/presentation/home/menu/sidebar/sidebar_new_page_button.dart:52]: Oh got it. I will make it 18 then.

**Review by richardshiue — APPROVED** [review]
LGTM

**Pull request merged**